data_structures/src/superblock.rs

```diff
@@ -422,19 +431,19 @@ mod tests {
     #[test]
     fn superblock_state_default_add_votes() {
         // When the superblock state is initialized to default (for example when starting the node),
-        // all the received superblock votes are marked as `MaybeValid` or `AlreadySeen`
+        // all the received superblock votes are marked as `NotInArs`
         let mut sbs = SuperBlockState::default();
```

Comment:
You could add a test similar to this one but using a non-empty bootstrapping committee. Check that for the first superblock only votes from members of the committee are accepted.

Reply:
great, doing that now